Marlin/temperature.cpp

```diff
@@ -1913,6 +1913,14 @@ ISR(TIMER0_COMPB_vect) {
       }
     }
   #endif //BABYSTEPPING
+
+    int freeMem = freeMemory();
+    if (freeMem < 400) {
```

Comment:
Please comment this. Lets test this using 400 and use 250 in the actual release.